.github/ISSUE_TEMPLATE/--first-timers-only.md

```diff
@@ -20,7 +20,7 @@ docs.
 
 ### A first-timer's Issue
 
-A [first-timers-only issue](https://github.com/nexB/scancode-toolkit/labels/%20first%20timers%20only)
+A [first-timers-only issue](https://github.com/nexB/scancode-toolkit/labels/first%20timers%20only)
 means we've worked to make it more legible to folks who either **haven't contributed to our codebase before,
```

Comment:
Why are you mixing in changes unrelated to this issue? Please make sure you have a clean PR that only has your code fixes!

Reply:
@pombredanne i tried to sign off my commit by running `git rebase HEAD~3 --signoff` Then somehow the commits got mixed. Please guide me about solving this.